core/src/main/scala/org/apache/spark/scheduler/MapStatus.scala

```diff
@@ -31,7 +31,7 @@ import org.apache.spark.util.Utils
 
 /**
  * Result returned by a ShuffleMapTask to a scheduler. Includes the block manager address that the
- * task ran on, the sizes of outputs for each reducer, and the number of outputs of the map task,
+ * task ran on, the sizes of outputs for each reducer, and the number of records of the map task,
```

Comment:
Shall we also change `the sizes of outputs for each reducer` to `the sizes of output records for each reducer`?

Reply:
size was about bytes; so it doesn't really matter whether it's a record or a row or a block. it's also already pointed out below that it's about bytes.

Reply:
As we are going to revert the sequence of prs, do we still need this?